### DISCUSSION TITLE
How to continue same test case after failing with one data and need to continue with other data.

### DISCUSSION THREAD

**Discussion by pardeeprj90** [cypress-io/cypress#9449] [Questions and Help]
I have a scenario in which I loaded test data in the fixture and now I am iterating that data while running the test case, but now with some data my test case assertion gets failed. in this case, I want to continue running the test case with other left data. how we can achieve this as in current if test case assertion gets failed then test case gets stopped and left data is not being used in the same test case.

**Comment by bahmutov** [comment]
Cypress will never allow you to continue a test when a test command or assertion fails. 

Instead, create multiple tests from your data as shown in the recipe "Dynamic tests" https://github.com/cypress-io/cypress-example-recipes#fundamentals - if the test fails for some data items, the other tests can continue.
